File: 3856-trim-trailing-vowels/3856-trim-trailing-vowels.ts
function v(ch: string): boolean {
    let str: string = "aeiou";
    return str.includes(ch);
}

function trimTrailingVowels(s: string): string {
    let n: number = s.length;
    let id: number = n - 1;

    while (id >= 0) {
        if (!v(s[id]))
            break;
        id--;
    }
    return s.slice(0, id + 1);
};

File: 3856-trim-trailing-vowels/3856-trim-trailing-vowels.go
func v(ch byte) bool {
	return ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u'
}

func trimTrailingVowels(s string) string {
    n := len(s)
    id := n - 1
    for id >= 0 {
        if !v(s[id]) {
            break
        }
        id -= 1
    }
    return s[: id + 1]
}